x-pack/plugins/observability_solution/inventory/public/components/entity_enablement/enable_entity_model_button.tsx

```diff
@@ -61,6 +62,7 @@ export function EnableEntityModelButton({ onSuccess }: { onSuccess: () => void }
         {i18n.translate('xpack.inventory.noData.card.button', {
           defaultMessage: 'Enable',
         })}
+        <TechnicalPreviewBadge style={{ marginTop: '4px' }} />
```

Comment:
Can't you define it in the `iconType` property on the EuiButton component and set iconSide="right"?

Reply:
I think you can try moving the <TechnicalPreviewBadge .../> component to `iconType` property. It looks like it accepts a react component:
```
import React, { PureComponent, ComponentType, SVGAttributes } from 'react';
...
export type IconType = EuiIconType | string | ComponentType;
```

Reply:
you're right, updating it

Reply:
https://github.com/elastic/kibana/pull/196353/commits/9f3ebabe3be029e55108dedf46b5e6c91efc4bbd